mitodl/edx-username-changer
# Bump sqlparse to 0.4.2

#### Pre-Flight checklist


#### What are the relevant tickets?
#8 

#### What's this PR do?
- Fixes a security alert
- Upgrades the package dependencies including `sqlparse=0.4.2`

#### How should this be manually tested?
- Install the `edx-username-changer` from this commit
- Follow the feature flag instruction from the Readme
- Once configurations are done, visit any user in Django admin and you should see the `username` as an `editable` field
- Check the same username is reflected in the `forum service` as well.

#### Where should the reviewer start?
- It would be a good idea to start with Readme and following the instructions added there.
